internal/plugins/helm/v1/api.go

```diff
@@ -136,19 +135,16 @@ func (p *createAPISubcommand) Run() error {
 
 // SDK phase 2 plugins.
 func (p *createAPISubcommand) runPhase2() error {
-	ogvk := p.createOptions.GVK
-	gvk := resource.GVK{Group: ogvk.Group, Version: ogvk.Version, Kind: ogvk.Kind}
-
 	// Initially the helm/v1 plugin was written to not create a "plugins" config entry
 	// for any phase 2 plugin because they did not have their own keys. Now there are phase 2
 	// plugin keys, so those plugins should be run if keys exist. Otherwise, enact old behavior.
 
 	if manifestsv2.HasPluginConfig(p.config) {
-		if err := manifestsv2.RunCreateAPI(p.config, gvk); err != nil {
+		if err := manifestsv2.RunCreateAPI(p.config, p.resource.GVK); err != nil {
```

Comment:
Could `p.resource` be nil?

Reply:
Only if `p.runPhase2()` is called directly, and since it is internal that should never happen. Regardless it's good practice to guard for that case.